src/core/Evaluate.re

```diff
@@ -22,23 +22,28 @@ type warning = {
 
 [@deriving show]
 type blockContent =
+  | BlockStart
   | BlockSuccess{
       msg: string,
       warnings: list(warning),
+      stdout: string,
     }
   | BlockError{
       error,
       warnings: list(warning),
+      stdout: string,
     };
 
 [@deriving show]
 type result =
   | Phrase{
       blockLoc: option(Loc.t),
       blockContent,
-      blockStdout: string,
     }
-  | Directive(string);
+  | Directive{
+		blockLoc: option(Loc.t),
```

Comment:
For now, this is always None. But it could be changed in the future. See #6

Reply:
Good to know, I noticed I never got it. I think I'll revert this change for now (I thought the `open`s were being bubbled up as `Directive`'s - but as you mentioned on Discord, it was another issue - I'll remove that line this PR too!)

Reply:
Removed this for now